pkgs/by-name/xw/xwax/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchurl, SDL, SDL_ttf, jack2, libmpg123, ffmpeg, cdparanoia }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xwax";
+  version = "1.9";
+
+  src = fetchurl {
+    url = "https://xwax.org/releases/xwax-${finalAttrs.version}.tar.gz";
+    hash = "sha256-m+2PoUMYKBhlA2H0kld1W/iR8UMWEGaqp7yoxszp9jI=";
+  };
+
+  buildInputs = [
+    # sdl
+    SDL
+    SDL_ttf
+
+    # audio server
+    jack2
+
+    # audio decoder
+    libmpg123
+    ffmpeg
+    cdparanoia
+  ];
+
+  configureFlags = [ "--enable-alsa" "--enable-jack" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp xwax  $out/bin/xwax
+  '';
```

Comment:
runHook {pre,post}Configure?

Reply:
> runHook {pre,post}Configure?

Thanks, didn't know about runHook, but i guess you meant `{pre|post}Install` ?